_posts/2016/2016-12-15-hr-ui.md

```diff
@@ -0,0 +1,100 @@
+I then get myself a glass of water and start working on my tasks.
+Every now and then I get "interrupted" by a [HipChat](https://www.hipchat.com/) message, which I respond to.
+Then, at 10:30 a.m. we have our daily, where the whole team comes together to talk about current tasks and eventual challenges.
+At 12 o'clock, we have lunch with the team, although I also enjoy to go and have lunch with other teams or bring my own food.
+Depending on whether we have meetings or not, I try to focus on my dedicated tasks in the afternoon.
+
+## What type of tasks are these?
+
+Currently, I'm working on a conceptual task.
+I'm helping out other teams to prepare UI elements, which can be reused and therefore I provide templates, for example, related to payment issues or alike.
+In particular, my colleagues from Jena and I jointly provide a design guideline for Team Ocean since they don't have their own designer.
+Other tasks are designing new features.
+Just yesterday, I designed new icons and I am redesigning our new drag and drop approach for the editor.
+My work is also about bug fixing, code reviews in HTML and LESS.
+
+## What are the required skills for an UI-Designer at ePages?
+
+Regarding soft-skills a UI-Designer should be communicative and shouldn't be afraid to ask colleagues, because we don't sit in only one office, but are spread out.
+I often communicate with my colleagues from Jena or Barcelona.
+Then, considering hard-skills, a UI-Designer needs to know HTML, Less, and should have an understanding of frontend technologies.
+Another requirement is to have some knowledge on UX, because we often work with the UX team in conceptional contexts.
+Also, it is absolutely necessary to be up-to-date when it comes to new design principles, patterns and trends.
+
+## What do you like at ePages and its company culture?
+
+I like the fact to be working in a team.
+Besides the daily meetings, I can structure my days as I like.
+Through ePages' arrangement of flexible working hours, I am able to come - as I mentioned before - a lot earlier to the office than my team colleagues and I am therefore a bit freer to organize my work day.
+Also, my colleagues are friendly and helpful.
+If I have a question, I can always count on their support.
+Concerning the culture of ePages, I appreciate our yearly get-together YOU a lot as well as the internal hackathons.
+I then get to see my colleagues from Jena and Barcelona.
+
+## Related posts
+
+* [Working as a Java Developer at ePages](https://developer.epages.com/blog/2016/03/01/hr-java.html)
+* [Working as a Ruby Developer at ePages](https://developer.epages.com/blog/2016/02/04/hr-ruby.html)
+* [Working as a QA Engineer at ePages](https://developer.epages.com/blog/2016/04/12/hr-quality-assurance.html)
+* [One month as an ePagee](https://developer.epages.com/blog/2015/08/11/one-month-as-an-epagee.html)
```

Comment:
Would remove this and the next one, but add these:
All of them in the correct chronology.

https://developer.epages.com/blog/2016/09/13/tech-recruiting.html
https://developer.epages.com/blog/2016/09/13/tech-recruiting.html